plugins/osdn/plugin.go

```diff
@@ -44,6 +64,26 @@ func CreatePlugin(registry *Registry, multitenant bool, hostname string, selfIP
 	return plugin, err
 }
 
+func isOsdnPlugin(pluginType string) bool {
+	switch strings.ToLower(pluginType) {
+	case SingleTenantPluginName:
+		return true
+	case MultiTenantPluginName:
+		return true
```

Comment:
L68-L72 can be simplified:
case SingleTenantPluginName, MultiTenantPluginName:
     return true